openfe/setup/ligand_network_planning.py

```diff
@@ -49,6 +50,8 @@ def generate_radial_network(ligands: Iterable[SmallMoleculeComponent],
       If no scorer is supplied, the first mapping provided by the iterable
       of mappers will be used.
     """
+    if isinstance(mappers, AtomMapper):
```

Comment:
having to repeat this is pretty ugly :/ could we not wrap it up into a decorator?

Reply:
nah, it makes it hard to read (the function is in two places).  It's two lines, it's fine

Reply:
Again, a call for Konnektor (or however the German spells it), where we could put the real work in a private method and the public method could handle this.